pkg/customercard/client.go

```diff
@@ -0,0 +1,125 @@
+package customercard
+
+import (
+	"context"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/internal/httpclient"
+)
+
+const (
+	urlBase        = "https://api.mercadopago.com/v1/customers/{customer_id}"
```

Comment:
é errado chamar de domain?
```suggestion
	domain        = "https://api.mercadopago.com/v1/customers/{customer_id}"
```